core/shared/platform/zephyr/zephyr_platform.c

```diff
@@ -199,11 +199,14 @@ os_mmap(void *hint, size_t size, int prot, int flags, os_file_handle file)
     return addr;
 }
 
+#if !defined(WASM_ENABLE_LIBC_WASI)
+/* os_mremap cause linking errors */
```

Comment:
Seems no need to change this file if `core/shared/platform/zephyr/shared_platform.cmake` doesn't add the two lines below?
```cmake
include (${CMAKE_CURRENT_LIST_DIR}/../common/memory/platform_api_memory.cmake)
set (source_all ${source_all} ${PLATFORM_COMMON_MEMORY_SOURCE})
```

Reply:
see above